jmx/src/main/java/org/jboss/as/jmx/PluggableMBeanServerImpl.java

```diff
@@ -1245,10 +1295,10 @@ boolean authorizeSensitiveOperation(String methodName, boolean readOnly, boolean
         return true;
     }
 
-    boolean authorizeSuperUserOrAdministrator(String methodName) throws MBeanException {
+    boolean authorizeSuperUserOrAdministrator(JmxTarget target , String methodName) throws MBeanException {
```

Comment:
Not caused by this PR, but this method should be private.

It should be renamed too, as the "SuperUserOrAdministrator" bit is an implementation detail of a particular rbac provider.